go/mysql/replication.go

```diff
@@ -164,3 +167,34 @@ func (c *Conn) SemiSyncExtensionLoaded() (SemiSyncType, error) {
 	}
 	return SemiSyncTypeOff, nil
 }
+
+func (c *Conn) BinlogInformation() (string, bool, bool, string, error) {
+	replicaField := c.flavor.binlogReplicaField()
+
+	query := fmt.Sprintf("select @@global.binlog_format, @@global.log_bin, %s, @@global.binlog_row_image", replicaField)
+	qr, err := c.ExecuteFetch(query, 1, true)
+	if err != nil {
+		return "", false, false, "", err
+	}
+	if len(qr.Rows) != 1 {
+		return "", false, false, "", errors.New("unable to read global variables binlog_format, log_bin, log_replica_updates, binlog_row_image")
```

Comment:
We're not using `replicaField` in the error message.

Reply:
Fixing!